### PR TITLE
Docs for smoke test

### PR DESCRIPTION
**Pull request by ruokun-niu**
- Added a page for smoke test under `reference`
  - Updated prereq
  - Updated commands to execute the smoke test
  - Updated smoke test overview
- Added a small section on the installation pages that direct the users to the smoke test page (which is optional)

### CODE REVIEW

File: docs/content/reference/smoke-test/_index.md
Comment (agentofreality) [inline docs/content/reference/smoke-test/_index.md:1]: This feels like How-To content and should be in the How-To Guides section.

Please put it in a new folder: How-To Guides/Test Drasi

File: docs/content/reference/smoke-test/_index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+type: "docs"
+title: "Smoke Test"
```
Comment (agentofreality) [inline docs/content/reference/smoke-test/_index.md:3]: Smoke Test a Drasi Installation

File: docs/content/reference/smoke-test/_index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+linkTitle: "Smoke Test"
+weight: 40
+description: >
+    Smoke test for Drasi Installation
```
Comment (agentofreality) [inline docs/content/reference/smoke-test/_index.md:7]: Learn how to quickly test a Drasi installation.

File: docs/content/reference/smoke-test/_index.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+{{< tabpane langEqualsHeader=true >}}
+{{< tab header="MacOS/VS Code Dev Container" lang="Bash" >}}
+curl -fsSL https://raw.githubusercontent.com/drasi-project/drasi-platform/main/dev-tools/smoke-tests/smoke-test.sh | /bin/bash -s <namespace>
+{{< /tab >}}
+{{< tab header="Windows PowerShell" lang="Bash" >}}
+Invoke-Command -ScriptBlock ([scriptblock]::Create([System.Text.Encoding]::UTF8.GetString((New-Object Net.WebClient).DownloadData('https://raw.githubusercontent.com/drasi-project/drasi-platform/main/dev-tools/smoke-tests/smoke-test.ps1')))) -ArgumentList <namespace>
+{{< /tab >}}
+{{< /tabpane >}}
+
+
```
Comment (agentofreality) [inline docs/content/reference/smoke-test/_index.md:40]: Need a section in here that describes what they will see when the run the test and it is successful, and what they might see of the test fails.

**Review by agentofreality — CHANGES_REQUESTED** [review]

File: docs/content/how-to-guides/smoke-test/_index.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
+---
+type: "docs"
+title: "Smoke Test a Drasi Installation"
+linkTitle: "Test Drasi"
```
Comment (agentofreality) [inline docs/content/how-to-guides/smoke-test/_index.md:4]:
```suggestion
linkTitle: "Smoke Test a Drasi Installation"
```

**Review by agentofreality — CHANGES_REQUESTED** [review]

**Review by agentofreality — APPROVED** [review]

**Comment by agentofreality** [comment]
@ruokun-niu lets not merge this. I have taken the changes into the doc work i am doing and will publish it with them.

**Comment by ruokun-niu** [comment]
Closing this PR

**Pull request closed without merging**